src/server/rdb_save.h

```diff
@@ -149,6 +149,22 @@ class SerializerBase {
 
   std::error_code WriteRaw(const ::io::Bytes& buf);
 
+  // Write journal entry as an embedded journal blob.
+  std::error_code WriteJournalEntry(std::string_view entry);
+
+  // Send FULL_SYNC_CUT opcode to notify that all static data was sent.
+  std::error_code SendFullSyncCut();
+
+  std::error_code WriteOpcode(uint8_t opcode) {
```

Comment:
since you are refactoring would you mind moving this and `SaveString` definition to `rdb_save.cc` for consistency ?